recipes/exprtk/all/conanfile.py

```diff
@@ -1,42 +1,51 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.layout import basic_layout
+from conan.tools.files import get, copy, load, save
 import os
-from conans import ConanFile, tools
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.53.0"
 
 
 class ExprTkConan(ConanFile):
     name = "exprtk"
     description = "C++ Mathematical Expression Parsing And Evaluation Library ExprTk"
-    license = ("MIT")
-    topics = ("exprtk", "cpp", "math", "mathematics", "parser", "lexer", "numerical")
+    license = "MIT"
+    topics = ("math", "mathematics", "parser", "lexer", "numerical", "header-only")
     homepage = "https://www.partow.net/programming/exprtk/index.html"
     url = "https://github.com/conan-io/conan-center-index"
-    settings = "compiler"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
```suggestion
    package_type = "header-library"
    settings = "os", "arch", "compiler", "build_type"
```

For conan v2, please define package_type.